.github/workflows/vast.yaml

```diff
@@ -893,21 +886,25 @@ jobs:
           # we attempt to restore from, which is the before sha for pushes and
           # the merge-base for pull requests.
           docker buildx build . -t tenzir/vast:latest --target production --load \
-            --build-arg VAST_BUILD_OPTIONS="${VAST_BUILD_OPTIONS}" \
+            --build-arg "VAST_BUILD_OPTIONS=${docker_build_args}" \
             --label "org.opencontainers.image.source=https://github.com/tenzir/vast" \
             --cache-to "type=registry,mode=max,ref=ghcr.io/tenzir/vast:${cache_to_ref}" \
             --cache-from "type=registry,ref=ghcr.io/tenzir/vast:${cache_to_ref}" \
             --cache-from "type=registry,ref=ghcr.io/tenzir/vast:${cache_from_ref}"
           docker buildx build . -t tenzir/vast-dev:latest --target development --load \
-            --build-arg VAST_BUILD_OPTIONS="${VAST_BUILD_OPTIONS}" \
+            --build-arg "VAST_BUILD_OPTIONS=${docker_build_args}" \
             --label "org.opencontainers.image.source=https://github.com/tenzir/vast" \
             --cache-from "type=registry,ref=ghcr.io/tenzir/vast:${cache_to_ref}" \
             --cache-from "type=registry,ref=ghcr.io/tenzir/vast:${cache_from_ref}"
           docker buildx build . -t tenzir/vast-deps:latest --target dependencies --load \
-            --build-arg VAST_BUILD_OPTIONS="${VAST_BUILD_OPTIONS}" \
+            --build-arg "VAST_BUILD_OPTIONS=${docker_build_args}" \
             --label "org.opencontainers.image.source=https://github.com/tenzir/vast" \
             --cache-from "type=registry,ref=ghcr.io/tenzir/vast:${cache_to_ref}" \
             --cache-from "type=registry,ref=ghcr.io/tenzir/vast:${cache_from_ref}"
+          if [[ "$GITHUB_EVENT_NAME" == "pull_request" ]]; then
+            docker tag tenzir/vast:latest "ghcr.io/tenzir/vast:${cache_to_ref}"
+            docker push "ghcr.io/tenzir/vast:${cache_to_ref}"
+          fi
```

Comment:
so, this doesn't push any extra layers, it just tags everything that was cached with buildx, right?